packages/dapi-grpc/protos/platform/v0/platform.proto

```diff
@@ -269,20 +270,16 @@ message GetDataContractsResponse {
 
 message GetDataContractHistoryRequest {
   bytes id = 1;
-  optional uint32 limit = 2;
-  optional uint32 offset = 3;
-  optional uint64 start_at_seconds = 4;
+  uint32 limit = 2;
+  uint32 offset = 3;
+  uint64 start_at_seconds = 4;
```

Comment:
why not ms?

Reply:
Yeah, I've just checked, it is indeed in ms, not seconds. I guess I just misspelled it somewhere and then continued to name it the same afterwards